scripts/devel/build_and_install.py

```diff
@@ -30,21 +30,16 @@
 from ros_buildfarm.workspace import ensure_workspace_exists
 
 
-def call_abi_checker(workspace_root, ros_version, env):
+def call_abi_checker(workspace_root, rosdistro_name, env):
     import rosdistro
 
-    condition_context = {}
-    condition_context['ROS_DISTRO'] = env['ROS_DISTRO']
-    condition_context['ROS_VERSION'] = ros_version
-    condition_context['ROS_PYTHON_VERSION'] = \
-        (env or os.environ).get('ROS_PYTHON_VERSION')
-    pkgs = get_packages_in_workspaces(workspace_root, condition_context)
+    pkgs = get_packages_in_workspaces(workspace_root)
     pkg_names = [pkg.name for pkg in pkgs.values()]
     assert pkg_names, 'No packages found in the workspace'
 
     # Filter packages in source space that has been released
     index = rosdistro.get_index(rosdistro.get_index_url())
-    dist_file = rosdistro.get_distribution_file(index, env['ROS_DISTRO'])
+    dist_file = rosdistro.get_distribution_file(index, rosdistro_name)
```

Comment:
out of curiosity, is it better to use `rosdistro_name` here instead of using the env value?

Reply:
I'm just avoiding passing a value from one spot in the script to another using an env variable, since the env variable was set using this value to begin with.

It seemed like a natural change - if there is a downside I'm not seeing, please point it out.